graphql/resolvers/PublicUser.js

```diff
@@ -1,11 +1,70 @@
 module.exports = {
-  async testimonial (user, args, {pgdb}) {
-    const testimonial = await pgdb.public.testimonials.findOne({userId: user.id})
+  async testimonial (user, args, { pgdb }) {
+    const testimonial = await pgdb.public.testimonials.findOne({
+      userId: user.id
+    })
     if (testimonial) {
       return {
         ...testimonial,
         name: user.name
       }
     }
+  },
+  async latestComments (user, args, { pgdb }) {
+    const userId = user.id
+    const limit = 10
```

Comment:
don't you want to get that over args?
latestComments(first/limit: Int) in schema?